src/i_winmusic.c

```diff
@@ -1238,6 +1252,22 @@ static void FillBuffer(void)
         return;
     }
 
+    if (player_state == STATE_STOPPED)
```

Comment:
This would be better with a switch so it's easy to see all states are handled and "playing" state does nothing by design.

Reply:
https://github.com/chocolate-doom/chocolate-doom/pull/1603/commits/bdd9e2c466cf8defc909d5bcc30fe7377bdf4816